lib/v8debugapi.js

```diff
@@ -331,6 +331,14 @@ module.exports.create = function(logger_, config_, fileStats_) {
               uncompiled + ' <<', err);
           }
         };
+      case 'es6':
```

Comment:
These extensions were pulled from the babel documentation. I don't know in practice which extensions people. We could implement setting breakpoints in compiled files where both source and target extensions are js but it may be worth waiting for that to be requested before adding the complication here.